src/icons/states/SandboxesInterstitialState.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import AccessibleSVG from '../accessible-svg';
+import { bdlBoxBlue, bdlBoxBlue10 } from '../../styles/variables';
+
+type Props = {
+    className?: string,
```

Comment:
If these have defaults set in the params should they still be `maybe` types?

Reply:
@mickr The other icons have `maybe` types for props with default values, so I followed that convention. Do you think I should change the types here?

Reply:
That is correct actually. Unless you use defaultProps, types need to have ?.
So hypothetically, if this was done at the bottom of the file

SandboxesInterstitialState.defaultProps = { className: '' }, then in the flow types you would not need a ? className: string

So current code is correct

Reply:
@sahiga thank you for the update, @priyajeet, in the future would it make sense to use `defaultProps` instead of using the default param in the de-structuring?